ui/src/lib/gql/user/useApi.ts

```diff
@@ -20,6 +31,30 @@ export const useUser = () => {
     };
   };
 
+  const useGetMeAndWorkspaces = (): GetMeAndWorkspaces => {
+    const { data, ...rest } = useGetMeAndWorkspacesQuery();
+    return {
+      me: data,
+      workspaces: data?.workspaces?.map((workspace) => ({
+        id: workspace.id,
+        name: workspace.name,
+        personal: workspace.personal,
+        members: workspace.members.map((member) => ({
+          userId: member.userId,
+          role: member.role,
+          user: member.user
+            ? {
+                id: member.user.id,
+                email: member.user.email,
+                name: member.user.name,
+              }
+            : undefined,
+        })),
+      })),
```

Comment:
I think you copy/pasted this from `createNewWorkspaceObject` found in the `workspace/useQueries.ts` file. Instead of just copy/pasting, please move the original function into gql/convert, and use that here and elsewhere.